config/settings_schema.json

```
@@ -516,21 +480,17 @@
         "type": "range",
         "id": "inputs_shadow_blur_radius",
         "min": 0,
-        "max": 20,
-        "step": 2,
+        "max": 40,
+        "step": 5,
         "unit": "px",
         "label": "Blur radius",
         "default": 0
       }
     ]
   },
   {
-    "name": "Text boxes",
-    "settings": [
-      {
-        "type": "header",
-        "content": "Typography"
-      },
+    "name": "Content containers",
+    "settings": [    
       {
         "type": "header",
         "content": "Borders"
```

```suggestion
        "content": "Border"
```
We have the singular form, like for `Shadow`.